### ISSUE TITLE
User icon and Admin icon should be marked as menu

### ISSUE THREAD

**Issue by mattab** [piwik/piwik#9553]
Goal of this issue is to improve accessibility of the top menu in Piwik to let users know that the "User icon (Personal settings)" and "Administration icon (Admin settings)" will open new menu structures.

Both the "User icon" and the "Settings icon", when clicked, will open a new left menu. To make this obvious to users with disabilities, we can mark these two icons/links with a special HTML attribute. 

Likely we need to add the `role` and `aria-label` attributes as in https://github.com/piwik/piwik/issues/9551

**Comment by mattab** [comment]
Not really needed now (we will soon remove the User Menu in https://github.com/piwik/piwik/issues/9354 )
